testCommon/src/testFixtures/java/com/virtuslab/gitmachete/testcommon/TestProcessUtils.java

```diff
@@ -22,15 +22,13 @@ public static String runProcessAndReturnStdout(Path workingDirectory, int timeou
     boolean completed = process.waitFor(timeoutSeconds, TimeUnit.SECONDS);
 
     String stdout = IOUtils.toString(process.getInputStream(), StandardCharsets.UTF_8);
-
+    String stderr = IOUtils.toString(process.getErrorStream(), StandardCharsets.UTF_8);
     String commandRepr = Arrays.toString(command);
 
-    if (!completed || process.exitValue() != 0) {
-      System.out.println("Stdout of ${commandRepr}: ${System.lineSeparator()}");
-      System.out.println(stdout);
-      System.err.println("Stderr of ${commandRepr}: ${System.lineSeparator()");
-      System.err.println(IOUtils.toString(process.getErrorStream(), StandardCharsets.UTF_8));
-    }
+    Assert.assertTrue("Stdout of " + commandRepr + " : " + System.lineSeparator() + " " + stdout,
+        !completed || process.exitValue() != 0);
+    Assert.assertTrue("Stderr of " + commandRepr + " : " + System.lineSeparator() + " " + stderr,
+        !completed || process.exitValue() != 0);
```

Comment:
i am feeling ashamed as i was checking for the wrong condition and now the build have been successful and testcase passed :(

Reply:
@PawelLipski  should i use the string interpolation instead of + and then push the code ?

Reply:
> @PawelLipski should i use the string interpolation instead of + and then push the code ?

Let's use `+` for now, I've got no idea why string interpolation didn't work (and don't want to debug it now 😅). Push the code indeed as you develop, to see if CI passes

Reply:
@PawelLipski okay so i am gonna append the line :28 and 30 assert message to the one assert message in the line 33 ? is it fine ?